components/package.json

```diff
@@ -41,10 +41,8 @@
     "node": "*"
   },
   "main": "dist/main.js",
+  "module": "src/index.js",
```

Comment:
I believe we could replace with with a generated es module in /dist and maintain the same auto-complete functionality while also being able to package only the dist folder (without src). Other libraries I've looked at manage to do that using Rollup, BUT.. Webpack doesn't support es modules yet https://github.com/webpack/webpack/issues/2933 but they might soon